backend/src/schema/resolvers/users.js

```diff
@@ -48,6 +48,29 @@ export default {
         throw new UserInputError(e.message)
       }
     },
+    blockedByPostAuthor: async (_parent, params, context, _resolveInfo) => {
```

Comment:
Why is it necessary that Alice can ask Bob if he blocked Mallory? Only Bob should be allowed to ask Mallory if she blocked him.

Reply:
We need to know if the `currentUser` has been blocked by the post author

Reply:
I don't understand... The mutation checks if the `currentUser` represented by `context.user.id` has been blocked by the post author, oder?